scout/constants/filters.py

```diff
@@ -23,3 +23,10 @@
     "region_annotations": ["exonic", "splicing"],
     "functional_annotations": SEVERE_SO_TERMS,
 }
+
+CLINICAL_FILTER_BASE_MEI = {
+    "variant_type": "clinical",
+    "region_annotations": ["exonic", "splicing"],
+    "functional_annotations": SEVERE_SO_TERMS_SV,
+    "swegen_freq": "0.02",
```

Comment:
Initial frequency to filter is up for discussion: not too high and not too low..

Reply:
I'm not sure about this value. I agree on 0.02